Makefile

```diff
@@ -53,21 +54,22 @@ check-licenses: fetch
 		"$(BUILDER_IMAGE)" \
 		bash -c "cargo deny --all-features check --disable-fetch licenses bans sources"
 
-# Builds, Lints and Tests the Rust workspace
-build: fetch check-licenses
+# Builds, Lints, and Tests the Rust workspace locally
+build: check-licenses
 	cargo fmt -- --check
-	cargo build --locked
 	cargo test --locked
+	cargo build --locked
 
-brupop-image: fetch check-licenses
+# Builds only the brupop image. Useful target for CI/CD, releasing, etc.
+brupop-image:
```

Comment:
brupop-image no longer has a dependency on fetch or check-licenses since both depend on the cargo / rust toolchain being available. This ensures that we can build the brupop image in an environment that may not have those tools.